xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variablesInit.vm

```diff
@@ -261,4 +261,8 @@
 @xwiki-icon-edit-section:             "\f040";
 @xwiki-icon-spinner:                  "$xwiki.getSkinFile('icons/xwiki/spinner.gif')";
 @xwiki-icon-spinner-height:           16px;
-@xwiki-icon-link-external:            "$xwiki.getSkinFile('icons/xwiki/external-link.png')";
\ No newline at end of file
+@xwiki-icon-link-external:            "$xwiki.getSkinFile('icons/xwiki/external-link.png')";
+
+## Standard pointer target sizes from WCAG recommendations
+@target-size-minimum:                 24px;
+@target-size-recommended:             44px;
```

Comment:
Wouldn't be better to put them directly in ``variables.less`` since you don't need Velocity to compute the default values?

Reply:
You're right, `variables.less` seems like a better place to put these :)